src/Motion.js

```diff
@@ -1,9 +1,9 @@
 export function managePerimeterCollision(state, obj){
   if(atLowerBound(state.screen, obj)){
-    state.currentScore += 10;
+    if(state.inGame){state.currentScore += 10}
     obj.gravity = false;
   } else if (atUpperBound(obj)) {
-    state.currentScore += 10;
+    if(state.inGame){state.currentScore += 10}
```

Comment:
_Disclaimer: Haven't pulled this down_ but does this actually add 10 to the current score?
- Curious given we aren't using `setState`